apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx

```diff
@@ -146,9 +146,9 @@ export class CohortViewTable extends React.Component {
           transforms: [sortable]
         },
         cell: {
-          format: status => {
-            return _.upperFirst(status);
-          },
+          format: status =>
+            ApplicationStatuses[this.props.viewType][status] ||
```

Comment:
I'm looking up the prettier labels [here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/application_dashboard/constants.js#L52) instead of just uppercasing the underscored value we get back from the server